packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/RunnerTests.swift

```diff
@@ -0,0 +1,12 @@
+//
```

Comment:
What is the purpose of this file?

Reply:
https://github.com/flutter/plugins/pull/6517#discussion_r990484765

Reply:
Turns out, the file is not necessary after all, see [this follow-up comment](https://github.com/flutter/plugins/pull/6517#discussion_r1007422985).